web/src/components/player/VideoPlayer.tsx

```diff
@@ -88,7 +88,7 @@ export default function VideoPlayer({
 
   return (
     <div data-vjs-player>
-      <div ref={videoRef} />
+      <div className="rounded-2xl overflow-hidden" ref={videoRef} />
```

Comment:
Chrome is super messy with whatever it's doing with videojs in this ref, but firefox and safari are fine. So odd